NGCHM/WebContent/javascript/lib/lexer/lexer-master/bower.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "lexer",
+  "main": "lexer.js",
+  "version": "1.7.9",
+  "homepage": "https://github.com/aaditmshah/lexer",
```

Comment:
Did this new dependency really come from this URL?

I notice the repo at that URL is archived, doesn't have any formal releases, and hasn't been touched in 7 years (other than to archive it 3 years ago)

Reply:
The README says it's installable via npm, which it is, but npm warns it's deprecated:

```bash
$ npm install lex
npm warn deprecated lex@1.7.9: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.
```

We really want to add this unsupported library to our code base?

Reply:
I guess it's not very big....